cicd/base/config.ini

```diff
@@ -0,0 +1,36 @@
+[DEFAULT]
+PRODUCT_COMPONENT = base
+PIPELINE_STACK_NAME = ala-${PRODUCT_NAME}-${PRODUCT_COMPONENT}-pipeline-${ENVIRONMENT}
+BASE_STACK_FILE_PFIX = base
+BASE_STACK_NAME = ala-${PRODUCT_NAME}-${PRODUCT_COMPONENT}-${ENVIRONMENT}
+AUTO_DEPLOY = false
+SLACK_DEPLOY_NOTIFICATION = false
+SLACK_CHANNEL = deployments
+DEFAULT_LOGOUT_REDIRECT_URL = https://www.ala.org.au
+
+[development]
+# code pipeline
+PIPELINE_STACK_NAME = ala-${PRODUCT_NAME}-${PRODUCT_COMPONENT}-pipeline-${CLEAN_BRANCH}
+BASE_STACK_NAME = ala-${PRODUCT_NAME}-${PRODUCT_COMPONENT}-${CLEAN_BRANCH}
+SLACK_DEPLOY_NOTIFICATION = true
+SLACK_CHANNEL = zabbix-userdetails
+AUTO_DEPLOY = true
+
+COGNITO_STACK_NAME = ala-cognito-pool-testing
+APP_CLIENT_DOMAIN = ${PRODUCT_NAME}-${CLEAN_BRANCH}.${HOSTED_ZONE}
+DEFAULT_LOGOUT_REDIRECT_URL = https://www-test.ala.org.au
+
+[testing]
+PIPELINE_STACK_NAME = ala-${PRODUCT_NAME}-${PRODUCT_COMPONENT}-pipeline-${CLEAN_BRANCH}
+BASE_STACK_NAME = ala-${PRODUCT_NAME}-${PRODUCT_COMPONENT}-${CLEAN_BRANCH}
+SLACK_DEPLOY_NOTIFICATION = true
+SLACK_CHANNEL = zabbix-userdetails
+AUTO_DEPLOY = true
+
+COGNITO_STACK_NAME = ala-cognito-pool-testing
+APP_CLIENT_DOMAIN = ${PRODUCT_NAME}-${CLEAN_BRANCH}.${HOSTED_ZONE}
```

Comment:
There should be an APP_CLIENT_DOMAIN in the [DEFAULT] section set to:
APP_CLIENT_DOMAIN = ${PRODUCT_NAME}-${ENVIRONMENT}.${HOSTED_ZONE}
Which will cover all non-development environments

then APP_CLIENT_DOMAIN can be removed from here

Reply:
Fixed